monai/bundle/workflows.py

```diff
@@ -226,6 +242,138 @@ def check_properties(self) -> list[str] | None:
         return [n for n, p in self.properties.items() if p.get(BundleProperty.REQUIRED, False) and not hasattr(self, n)]
 
 
+class PythonicWorkflow(BundleWorkflow):
+    """
+    Base class for the pythonic workflow specification in bundle, it can be a training, evaluation or inference workflow.
+    It defines the basic interfaces for the bundle workflow behavior: `initialize`, `run`, `finalize`, etc.
+    And also provides the interface to get / set public properties to interact with a bundle workflow.
```

Comment:
```suggestion
    This also provides the interface to get / set public properties to interact with a bundle workflow through defined `get_<property>` accessor methods or directly defining members of the object.
```
There should be something said about this behaviour which is only seen in the body of `_get_property`.